.github/workflows/benchmark.yml

```diff
@@ -15,13 +15,13 @@ jobs:
         shell: bash -l {0}
         run: |
           mamba env update --quiet -n test -f libflatsurf/environment.yml
-          mamba install --quiet --yes asv
```

Comment:
A working (experimental) version is now installed by the `flatsurf/actions/asv` action.